js/contextMenus.js

```diff
@@ -134,6 +135,11 @@ function tabTemplateInit (frameProps) {
     })
   }
 
+  items.push(Object.assign({},
+    CommonMenu.reopenLastClosedTabItem,
+    { enabled: WindowStore.getState().get('closedFrames').size > 0 }
```

Comment:
I'm not a fan of including windowStore here, in favor of instead passing in info to contextrMenu but this is ok for now. Thanks.